<ui version="4.0" >
 <class>CreateMidasItemDialog</class>
 <widget class="QDialog" name="CreateMidasItemDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>382</width>
    <height>422</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Create item</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="MIDASDesktopIcons.qrc" >
    <normaloff>:/icons/document_add.png</normaloff>:/icons/document_add.png</iconset>
  </property>
  <property name="modal" >
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="0" column="0" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>Authors</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="margin" >
      <number>3</number>
     </property>
     <property name="buddy" >
      <cstring>authorsEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <layout class="QHBoxLayout" name="horizontalLayout_2" >
     <item>
      <widget class="QLineEdit" name="authorsEdit" >
       <property name="toolTip" >
        <string>First and Last name should be seperate using a &lt;i>comma&lt;/i>.&lt;br> Different Authors {First, Last} name pairs should be seperated using a &lt;i>semicolon&lt;/i>.</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="addAuthorButton" >
       <property name="icon" >
        <iconset resource="MIDASDesktopIcons.qrc" >
         <normaloff>:/icons/add2.png</normaloff>:/icons/add2.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="label_4" >
     <property name="text" >
      <string>Title </string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="margin" >
      <number>3</number>
     </property>
     <property name="buddy" >
      <cstring>titleEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="QLineEdit" name="titleEdit" />
   </item>
   <item row="2" column="0" >
    <widget class="QLabel" name="label_6" >
     <property name="text" >
      <string>Keyword</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="margin" >
      <number>3</number>
     </property>
     <property name="buddy" >
      <cstring>keywordsEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1" >
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <item>
      <widget class="QLineEdit" name="keywordsEdit" >
       <property name="toolTip" >
        <string>Different keyword should separated using a &lt;i>slash&lt;/i></string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignLeading</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="addKeywordButton" >
       <property name="icon" >
        <iconset resource="MIDASDesktopIcons.qrc" >
         <normaloff>:/icons/add2.png</normaloff>:/icons/add2.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0" >
    <widget class="QLabel" name="label_2" >
     <property name="text" >
      <string>Date</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="margin" >
      <number>3</number>
     </property>
     <property name="buddy" >
      <cstring>issueDateEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1" >
    <widget class="QDateEdit" name="issueDateEdit" >
     <property name="displayFormat" >
      <string>MMM yyyy</string>
     </property>
     <property name="calendarPopup" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0" >
    <widget class="QLabel" name="label_9" >
     <property name="text" >
      <string>Description</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
     </property>
     <property name="margin" >
      <number>3</number>
     </property>
     <property name="buddy" >
      <cstring>descriptionEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="1" >
    <widget class="QTextEdit" name="descriptionEdit" >
     <property name="maximumSize" >
      <size>
       <width>16777215</width>
       <height>120</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="5" column="0" >
    <widget class="QLabel" name="label_7" >
     <property name="text" >
      <string>Abstract</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
     </property>
     <property name="margin" >
      <number>3</number>
     </property>
     <property name="buddy" >
      <cstring>abstractEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="1" >
    <widget class="QTextEdit" name="abstractEdit" >
     <property name="maximumSize" >
      <size>
       <width>16777215</width>
       <height>120</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="6" column="0" >
    <widget class="QLabel" name="label_8" >
     <property name="text" >
      <string>Sponsor</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
     </property>
     <property name="margin" >
      <number>3</number>
     </property>
     <property name="buddy" >
      <cstring>sponsorEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="6" column="1" >
    <widget class="QTextEdit" name="sponsorEdit" >
     <property name="maximumSize" >
      <size>
       <width>16777215</width>
       <height>120</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="7" column="0" >
    <widget class="QLabel" name="label_3" >
     <property name="text" >
      <string>Links</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="margin" >
      <number>3</number>
     </property>
     <property name="buddy" >
      <cstring>linksEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="7" column="1" >
    <widget class="QLineEdit" name="linksEdit" />
   </item>
   <item row="8" column="1" >
    <widget class="QCheckBox" name="saveEntryCheckBox" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="text" >
      <string>Save Entries</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0" colspan="2" >
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="MIDASDesktopIcons.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CreateMidasItemDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>196</x>
     <y>465</y>
    </hint>
    <hint type="destinationlabel" >
     <x>196</x>
     <y>243</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CreateMidasItemDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>196</x>
     <y>465</y>
    </hint>
    <hint type="destinationlabel" >
     <x>196</x>
     <y>243</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
